jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -104,6 +104,14 @@ public void buildDocker()
     Preconditions.checkNotNull(jibExtension);
     TaskCommon.disableHttpLogging();
 
+    if (jibExtension.getUseOnlyProjectCache()) {
+      getLogger()
+          .warn(
+              "jib.useOnlyProjectCache is deprecated; consider using the 'jib.useOnlyProjectCache' "
```

Comment:
Setting the `jib.useOnlyProjectCache` config will have no effect, right? Then, I'd say "use the ... system property" instead of "consider using ..."

Also, what's the correct system property name, `useOnlyProjectCache` or `useProjectOnlyCache`?

Reply:
Yeah it's `useOnlyProjectCache`, I messed up haha